src/y0/algorithm/identify/cg.py

```diff
@@ -308,7 +294,7 @@ def make_counterfactual_graph(
                         cf_graph, node_at_intervention1, node_at_intervention2
                     )
                     if is_inconsistent(new_event, node_at_intervention1, node_at_intervention2):
-                        # TODO needs test case
+                        # FIXME needs test case
```

Comment:
TEST CASE

it's not clear what chaos this will cause when running the ID* and IDC* - therefore we need both a unit test for this directly and also one with the high-level algorithms that covers this